packages/contracts/deploy/L1/019-L1GovernanceToken.ts

```diff
@@ -14,13 +13,16 @@ const deployFn: DeployFunction = async (hre) => {
     'L1StandardBridgeProxy'
   )
 
-  const l1MintManagerAddress = await getDeploymentAddress(hre, 'L1MintManager')
+  const Artifact__GovernanceTokenProxy = await hre.companionNetworks[
```

Comment:
This means that `GovernanceTokenProxy` should be deployed on L2 first? Or, why don't we get the address of deterministic token contract from [here](https://github.com/kroma-network/kroma/pull/333/files#diff-120093f03b10b9455e845a77c01950510e03c25c7133d79c3d1a3aed678c421cR200)?

Reply:
Then I'll just use `L1GovernanceTokenProxy` address instead of the L2 one.